racket/src/racket/gc2/Makefile.in

```diff
@@ -502,6 +502,11 @@ foreign.@LTO@: $(XSRCDIR)/foreign.c
 main.@LTO@: $(XSRCDIR)/main.c
 	$(CC) $(ALL_CFLAGS) -c $(XSRCDIR)/main.c -o main.@LTO@
 
+gmp_alpha_gcc.@LTO@: $(srcdir)/gmp/gmp_alpha_gcc.s
```

Comment:
Should the `$(srcdir)`s be `$(SRCDIR)` or `$(srcdir)/../src` ?

Reply:
Correct! The rules are broken - fixing it now.